src/main/g8/build.sbt

```diff
@@ -1,3 +1,16 @@
-scalaVersion := "3.1.3"
+scalaVersion := "3.2.2"
 
 enablePlugins(ScalaNativePlugin)
+
+// set to Debug for compilation details (Info is default)
+logLevel := Level.Info
+
+// import to add Scala Native options
+import scala.scalanative.build._
+
+// defaults set with common options shown
+nativeConfig ~= { c =>
```

Comment:
I like the way you handled the `nativeConfig ~=` section and will probably use the snippet in my own local.sbt.

Reply:
I want to make the next steps or experimentation easier. I appreciate the 👍